apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -17,13 +17,28 @@ const ADD_STUDENT_LEVEL_PROGRESS = 'sectionProgress/ADD_STUDENT_LEVEL_PROGRESS';
 
 // Action creators
 export const setScriptId = scriptId => ({ type: SET_SCRIPT, scriptId});
-export const setSection = section => ({ type: SET_SECTION, section });
 export const setValidScripts = validScripts => ({ type: SET_VALID_SCRIPTS, validScripts });
 export const setCurrentView = viewType => ({ type: SET_CURRENT_VIEW, viewType });
 export const addScriptData = (scriptId, scriptData) => ({ type: ADD_SCRIPT_DATA, scriptId, scriptData });
 export const addStudentLevelProgress = (scriptId, studentLevelProgress) => ({
   type: ADD_STUDENT_LEVEL_PROGRESS, scriptId, studentLevelProgress
 });
+export const setSection = (section) => {
+  // Sort section.students by name.
+  const sortedStudents = section.students.sort((studentA, studentB) => {
+    // Ignore case.
+    const studentNameA = studentA.name.toLowerCase();
+    const studentNameB = studentB.name.toLowerCase();
```

Comment:
I'm not familiar with the pros/cons but would it be worth using `localeCompare` instead?

Reply:
Seems better tested and shorter, so I'll go with it 👍